src/legacy/server/config/schema.js

```diff
@@ -244,8 +244,17 @@ export default () => Joi.object({
         }))
       })).default([])
     }).default(),
-    manifestServiceUrl: Joi.string().default('https://catalogue.maps.elastic.co/v7.0/manifest'),
+    manifestServiceUrl: Joi.string().default('https://catalogue-staging.maps.elastic.co/v7.2/manifest'),
     emsLandingPageUrl: Joi.string().default('https://maps.elastic.co/v7.0'),
+    emsTileLayerId: Joi.object({
+      bright: Joi.string().default('road_map'),
+      desaturated: Joi.string().default('road_map_desaturated'),
+      dark: Joi.string().default('dark_map'),
+    }).default({
+      bright: 'road_map',
```

Comment:
reference this in these locations as well

https://github.com/thomasneirynck/kibana/blob/007de2e1d20ba0b771a45bb385295ef6e3eed528/src/legacy/core_plugins/tile_map/public/base_maps_visualization.js#L148

https://github.com/elastic/kibana/pull/36074/files#diff-f6c8dc76d602dbf9ec17cc7df9004b23R39

Reply:
reference config here too: https://github.com/thomasneirynck/kibana/blob/007de2e1d20ba0b771a45bb385295ef6e3eed528/src/legacy/core_plugins/vega/public/vega_view/vega_map_view.js#L39